examples/run_CPMG.py

```diff
@@ -31,31 +30,25 @@
 # {{{set phase cycling
 phase_cycling = True
 if phase_cycling:
-    ph1_cyc = r_[0, 1, 2, 3]
-    nPhaseSteps = 4
+    ph1_cyc = array([(j*2+k)%4 for k in range(4) for j in range(2)])
+    ph2_cyc = array([(k+1)%4 for k in range(4) for j in range(2)])
+    nPhaseSteps = 8
```

Comment:
no: (1) are you sure you need phase steps and (2) why isn't it len of previous???

Reply:
see lines 55-58 where I verify the number of total points using nPhaseSteps - not sure what you are asking for #2